tests/native/sender.sh

```diff
@@ -0,0 +1,19 @@
+set -e
+if [ $4 == "relay "]
```

Comment:
```suggestion
if [ $4 == "relay"]
```
Is this extra space needed?

Reply:
No, good spot. 👍🏼